### PR TITLE
feat(breadcrumbs): add breadcrumbs component

### PR DESCRIPTION
**Pull request by dine**

### CODE REVIEW

File: src/components/ui-breadcrumbs.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <ul v-if="items.length > 0" class="flex overflow-x-scroll">
+        <li
+            v-for="(item, index) in items"
+            :key="index"
+            :class="{ 'text-dark-3': index === items.length - 1 }"
+        >
+            <span
+                v-if="index === items.length - 1"
```
Comment (Zishrodrigues) [inline src/components/ui-breadcrumbs.vue:9]: This check is done twice in the template. Consider putting it in a `computed` prop.

**Review by Zishrodrigues — APPROVED** [review]

File: src/components/ui-breadcrumbs.vue (hunk 1 above)
Comment (jpsc) [reply src/components/ui-breadcrumbs.vue:9]: Also you can use `items.at(-1)`. [.at MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)

File: src/components/ui-breadcrumbs.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                    whitespace-nowrap
+                    align-middle
+                    cursor-pointer
+                    items-center
+                    focus:outline-none
+                    text-dark-2
+                    hover:text-dark-1
+                "
+                :href="item.href"
+                >{{ item.label
```
Comment (jpsc) [inline src/components/ui-breadcrumbs.vue:30]: Do we need to care about long text by using css truncate or something similar?

File: src/components/ui-breadcrumbs.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                }}<IconArrowRight
+                    role="presentation"
```
Comment (jpsc) [inline src/components/ui-breadcrumbs.vue:32]: The svg itself already includes a `role="img"` so I'm not sure if this is valid.

**Comment by jpsc** [comment]
I'm not sure if this component has the correct API. The way that is built doesn't allow for the use of a `<NuxtLink>` component. I would argue that a big part of our breadcrumbs would work inside of the same `Nuxt` application.

**Pull request closed without merging**